src/architect/environment/environment.utils.ts

```diff
@@ -25,14 +25,20 @@ export class EnvironmentUtils {
     }),
   };
 
-  static async getEnvironment(api: AxiosInstance, account: Account, environment_name?: string): Promise<Environment> {
+  static async getEnvironment(api: AxiosInstance, account: Account, environment_name?: string, strict = true): Promise<Environment> {
```

Comment:
Instead of adding another arg to the function, can you please create an options type to pass in that would look something like the following:

```ts
{
  environment_name?: string;
  strict?: boolean;
}
```

Ideally we can do that to keep the number of function args low.

Reply:
added new interface, and updated all calls to this function to use the new signature.

```typescript
export interface GetEnvironmentOptions {
  environment_name?: string;
  strict?: boolean;
}
```